src/EditorFeatures/Core/Interactive/InteractiveSession.cs

```diff
@@ -194,6 +199,18 @@ private void AddSubmissionProjectNoLock(ITextBuffer submissionBuffer, string lan
             _currentSubmissionProjectId = ProjectId.CreateNewId(newSubmissionProjectName);
             var newSubmissionDocumentId = DocumentId.CreateNewId(_currentSubmissionProjectId, newSubmissionProjectName);
 
+            // Associate a name with both the editor document and our roslyn document so LSP can make requests on it.
+            var extension = languageName switch
```

Comment:
>extension [](http://example.com/codeflow?start=16&length=9)

Would be better to add `Extension` property to `InteractiveEvaluatorLanguageInfoProvider` and then use `_languageInfo.Extension`